weasyl/comment.py

```diff
@@ -1,3 +1,4 @@
+# encoding: utf-8
```

Comment:
Was any special character added in the file? I could be mistaken, but I only see standard ASCII, and no Unicode that would require this addition.
```suggestion
```

Reply:
I used an rsquo here:

```python
# the parent comment isn’t visible to this user
```

Reply:
Ah! Gotcha. I didn't see that hiding in there.